package-res/resources/web/dojo/pentaho/common/overrides/dojo/_base/kernel.js

```diff
@@ -18,7 +21,7 @@ define(["../has", "./config", "require", "module"], function(has, config, requir
 
 			// notice dojo takes ownership of the value of the config module
 			config:config,
-			global:window,
+			global:global,
 			dijit:dijit,
```

Comment:
`window` is not defined in node.js, while global has the correct value in node.js as well.